.github/workflows/integration.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-yarn-
       - name: install dependencies
-        run: yarn install --frozen-lockfile --prefer-offline
+        run: yarn install --frozen-lockfile --prefer-offline && yarn run postinstall
```

Comment:
Just out of curiosity: Why is this required here but not locally?

Reply:
No idea, it's still not applying the patch, I'll just disable the test now to be able to debug the next steps further